### ISSUE TITLE
Would a POV type export be available for sound position s in soundmanager?

### ISSUE THREAD

**Issue by richclarkee** [vpinball/vpinball#639]
**Everytime you update a table you must reposition andy changes in sound manager**
If a sound position [sound manager] export would be available that would be aces [...]

**export sound manager positions**

**Describe alternatives you've considered**

**sorry if ive done this wrong...total noob to github, sent here by the legend Rajo Joey**

**Comment by emb417** [comment]
I appreciate this request. There are some tables where small tweaks to the volume of certain sounds or table/backglass position are wanted to optimize for my cabinet speaker setup. As new versions of the table come out, those settings are lost. Would be nice to have an export to the table.ini.

**Comment by richclarkee** [comment]
Indeed, i appreciate that not all tables are the same so to match would be a problem, BUT....as we now move into a more standardasied nfozzy, fleep situation i fee lit could work, or even if it was saved in the table .vpx file like the old days of pov?
